ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json

```diff
@@ -1,4 +1,18 @@
 [
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "taskList",
+    "UserRole": "[LASOLICITOR]",
```

Comment:
I doubt it but just want to check because I always confuse myself. Do we need to add permissions to the base idam solicitor role or will the case role be enough?

Reply:
case role is enough

Reply:
if it uses idam-role the problem is as follow:
there is user with caseworker-publiclaw-solicitor which belongs to different organisation,
this user is added via manage reps to a case, nut not as a LA
then this user will see task list, but should not